app/tables/capital-projects/CapitalProjectsTable.jsx

```diff
@@ -61,7 +64,8 @@ const filterAndSortData = (data, filterBy, colSortDirs) => {
 };
 
 
-class CPTable extends React.Component { // eslint-disable-line
+class CPTable extends React.Component {
+  // eslint-disable-line
```

Comment:
Could you add the specific rule that needs to be disabled? I find it more precise in its execution and expressive to the next developer reading the code.

Reply:
I don't know which specific rule was disabled here. It's presence pre-dates my work. I'll look into it.

Reply:
oh yeah, this is in the diff because of linting. My eye caught it and forgot that it's not actually your change. So... nevermind; I don't want to make someone responsible for code they didn't actually touch.

Reply:
You know what? I deleted the line and there was no warning shown. What the heck?